### PR TITLE
Remove any SPN ID logging done from azure CLI 

### PR DESCRIPTION
**Pull request by rjaini**
Remove any SPN ID logging done from azure CLI

### CODE REVIEW

File: kubernetes/template/DeploymentTemplates/script.sh [hunk 1]
```diff
@@ -380,6 +380,6 @@ log_level -i "ARM template generated at $PWD/_output/$MASTER_DNS_PREFIX director
 cd $PWD/_output/$MASTER_DNS_PREFIX 
 
 log_level -i "Deploy the template."
-az group deployment create -g $RESOURCE_GROUP_NAME --template-file azuredeploy.json --parameters azuredeploy.parameters.json > /dev/null
+az group deployment create -g $RESOURCE_GROUP_NAME --template-file azuredeploy.json --parameters azuredeploy.parameters.json --output none
```
Comment (jadarsie) [inline kubernetes/template/DeploymentTemplates/script.sh]: I think that our documentation is pointing people to check this log file for troubleshooting purposes.
Maybe it is worthwhile to delay this change.

**Comment by jadarsie** [comment]
Is it not OK to log SPN ID?

File: kubernetes/template/DeploymentTemplates/script.sh (hunk 1 above)
Comment (rjaini) [reply kubernetes/template/DeploymentTemplates/script.sh]: None format will only suppress the return success message and not errors and warnings.
The details are mentioned below:
https://docs.microsoft.com/en-us/cli/azure/format-output-azure-cli?view=azure-cli-latest

**Comment by rjaini** [comment]
> Is it not OK to log SPN ID?

In the current format its not as in case if customer has multiple SPN it will be confused. Ideally if we want then we should log it in the parameter section.

**Pull request merged**